File: Support Material/Demofiles/M14/CS/CustomerManagement/App_Code/Customer.cs
using System.ComponentModel.DataAnnotations;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace CustomerManagementModel
{
    [ScaffoldTable(true)]
    public class CustomerMetadata
    {
        [ScaffoldColumn(false)]
        public object ID;
        [ScaffoldColumn(false)]
        public object CreatedDate;
        [ScaffoldColumn(false)]
        public object CreatedBy;
        [ScaffoldColumn(false)]
        public object ModifiedDate;
        [ScaffoldColumn(false)]
        public object ModifiedBy;
    }

    /// <summary>
    /// Summary description for Customer
    /// </summary>
    [MetadataType(typeof(CustomerMetadata))]
    public partial class Customer
    {

    }
}

File: Support Material/Demofiles/M13/VB/CustomerManagement/App_Code/Customer.vb
Imports Microsoft.VisualBasic
Imports System.ComponentModel.DataAnnotations

Namespace CustomerManagementModel
    <ScaffoldTable(True)>
    Public Class CustomerMetadata
        <ScaffoldColumn(False)>
        Public Property ID As Object
        <ScaffoldColumn(False)>
        Public Property CreatedDate As Object
        <ScaffoldColumn(False)>
        Public Property CreatedBy As Object
        <ScaffoldColumn(False)>
        Public Property ModifiedDate As Object
        <ScaffoldColumn(False)>
        Public Property ModifiedBy As Object
    End Class

    <MetadataType(GetType(CustomerMetadata))>
    Partial Public Class Customer

    End Class
End Namespace